src/main/java/com/org/jenkins/custom/jenkins/distribution/service/PackagerController.java

```diff
@@ -35,15 +41,25 @@
         }
     }
 
+    /**
+     * @param postPayload Configuration for the war file
+     * @return War file
```

Comment:
Let's fix this javadoc. The thing being returned is not the war file, it is a ResponseEntity. Now please correct me if I am wrong, but I think this response entity has a body whose sole content is the war file in the form of a byte array of some kind. Please clarify that in the javadoc.

Reply:
Clarified :)